d3-scripts/src/d3_scripts/yaml_tools.py

```diff
@@ -59,6 +60,26 @@ def load_claim(file_name: str):
     return yaml_data
 
 
+@functools.lru_cache(maxsize=None)
+def __yaml_lint_config():
+    """YAML linting config. Cached, since it's slow."""
+    return YamlLintConfig(
+        r"""
+            extends: default
+            rules:
+                document-start:
+                    # so we don't need to start YAML files with ---
+                    present: false
+                line-length:
+                    # 80 characters is too small for 1080p/4K monitors
+                    max: 120
+                indentation:
+                    spaces: consistent
+                    indent-sequences: consistent
+        """
+    )
```

Comment:
Do you need to cache this if it creates the same type of config each time? If it's only defined once couldn't you store the config as a variable in the upper scope 
```suggestion
_yaml_lint_config = YamlLintConfig(
    r"""
        extends: default
        rules:
            document-start:
                # so we don't need to start YAML files with ---
                present: false
            line-length:
                # 80 characters is too small for 1080p/4K monitors
                max: 120
            indentation:
                spaces: consistent
                indent-sequences: consistent
    """
)
```

Reply:
That's fair

Reply:
If the performance benefit is lost then definitely keep it